stko/molecular/networkx/network.py

```diff
@@ -14,6 +14,92 @@
 logger = logging.getLogger(__name__)
 
 
+class PositionedAtom:
+    """
+    A container for stk.Atom and a position.
+
+    """
+
+    def __init__(self, atom, position):
+        """
+        Initialize a :class:`PositionedAtom`.
+
+        Parameters
+        ----------
+        atom : :class:`stk.Atom`
+            The atom.
+
+        position : : :class:`tuple` of :class:`float`
+            The position of the atom.
```

Comment:
Can you elaborate on this? e.g. of the form x,y,z corresponding to the coordinates of the system